src/Phansible/Roles/Xdebug.php

```diff
@@ -34,4 +36,17 @@ public function getInitialValues()
           'settings' => [],
         ];
     }
+
+    public function transformValues(array $values, VagrantBundle $vagrantBundle)
+    {
+        if ($values['install'] == 1) {
+            $values['settings'] = [
```

Comment:
Why not setting this in `getInitialValues` and adding them as hidden in the forms?

`RoleValuesTransformer` right now is a patch for very selected special cases and I wouldn't like to spread that to other parts of the system.